### DISCUSSION TITLE
HC Dungeons bug list

### DISCUSSION THREAD

**Discussion by balleny** [chromiecraft/chromiecraft#5802] [General]
### Reported issues

- https://github.com/chromiecraft/chromiecraft/issues/5741
- https://github.com/chromiecraft/chromiecraft/issues/5792
- https://github.com/chromiecraft/chromiecraft/issues/5797
- https://github.com/chromiecraft/chromiecraft/issues/5796
- https://github.com/chromiecraft/chromiecraft/issues/5794
- https://github.com/chromiecraft/chromiecraft/issues/5790
- https://github.com/chromiecraft/chromiecraft/issues/5795
- https://github.com/chromiecraft/chromiecraft/issues/5734
- https://github.com/chromiecraft/chromiecraft/issues/5737
- https://github.com/chromiecraft/chromiecraft/issues/5801
- https://github.com/chromiecraft/chromiecraft/issues/5779
- **21st of June:** https://github.com/chromiecraft/chromiecraft/issues/5811
- **22th of June:** https://github.com/chromiecraft/chromiecraft/issues/5822
- **1st of July:** https://github.com/chromiecraft/chromiecraft/issues/5856
- **4th of July** https://github.com/chromiecraft/chromiecraft/issues/5872 (the name sucks, but see my comment about the actual bug)
- **22nd of July** https://github.com/chromiecraft/chromiecraft/issues/5956
- https://github.com/chromiecraft/chromiecraft/issues/5540
- https://github.com/chromiecraft/chromiecraft/issues/5558

### Bugs not yet reported, fully researched etc.

- Arcatraz HC Sentinals are rootable
- Manatombs additional boss YOR misses all immunities
- Mana Tombs first boss ability Void Blast isnt cast / cast only after a long delay
- Is MT second boss using his heroic-only ability "Arcing Smash"?
- Manatombs final boss: the Ethereal Beacons should convert into Ethereal Apprentice after a while if not killed. havent seen this happen yet (both normal and heroic)
- ~~first boss in Blood Furnance (The Maker) seems to use Throw Beaker ability on tank only, not on random person~~
- waves on second boss in Blood Furnance (Broggok) should be additionally on a timer in heroic (2 min afaik) (its both, time based and wave start after previous wave cleared)
- Wrath-Scryer Soccothrates in Arca doesnt use Charge and Flamewave properly in combination. He should charge a random player (tank included) and the path charging should become a trail of fire.  Instead he drops stacking flames in a spot and charges separately. Also, the fire isnt considered AoE but direct damage (kills pets)
- Dalliah the Doomsayer in Heroic Arcatraz should cast Shadow Wave on the player the furthest away
- door behind second boss in Shadow Labs HC is missing -> https://github.com/chromiecraft/chromiecraft/issues/5855
- ~~several bosses are level 73 instead of 72~~ fixed

### Loot bugs (mostly resolved)
- ~~Mecha has the wrong cache not granting Badges~~ fixed
- ~~Old Hillsbrad drops normal mode loot~~ fixed
- **23th of June:** Avatar in AC has a variable amount of items dropping (0 to 3 observed thus far) instead of a fixed number of items dropping

**Comment by Gultask** [comment]
> Manatombs additional boss YOR misses all immunities

Fixed

> door behind second boss in Shadow Labs HC is missing

Vorpil or Blackheart?
If Blackheart it's because we had to remove it for a while and I may have forgot to readd it, if Vorpil then the door is there, but the proper fix can't be made without major core improvements allowing serverside areatriggers.

- [[Steamvault HC] Mekgineer spawns to few adds](https://github.com/chromiecraft/chromiecraft/issues/5790)

See discussion here: https://github.com/azerothcore/azerothcore-wotlk/pull/16580#issuecomment-1595895107

- [[Shadowlabs HC] Grandmaster Vorpil interruptable](https://github.com/chromiecraft/chromiecraft/issues/5795)

Fixed

**Reply by balleny** [reply]
Blackhearts door is missing.

**Reply by Gultask** [reply]
![image](https://github.com/chromiecraft/chromiecraft/assets/100873791/9ee31ade-03a1-47af-8dd2-f980486c1bdb)

**Reply by balleny** [reply]
door is missing.

**Comment by Paradigmxd** [comment]
Hi - to add to this 

1. Ramps Hc last boss is completely broken - the drake does not spit fire while in the air; The drake does not breathe fire and doesnt put fire on the ground whilst fighting on the ground
2. Exarch in Auchenai Crypts - The room is not linked, meaning peopel can skip half the adds and fight the boss; B) The Split disappears after Exarch is Defeated...
3. In Shadow Labs - the last room is linked inappropriatelly, when player pulls the first 2 mobs the whole room (almost and literary) comes to fight, making this impossible to finish unless FD/Vanish pulls are used
4. The Skeletons in Auchenai Crypts before the last boss are all linked, excluding 1st pack.

**Reply by balleny** [reply]
1. should be covered with https://github.com/chromiecraft/chromiecraft/issues/5792
2. true that is an issue for normal and heroic mode
3. havent seen this occur yet but guildmates mentioned similar chaining issues
4. its 3 independent packs. but the 2nd and 3rd pack are so close that they chain on Chromie

thats my knowledge about your points

**Reply by Gultask** [reply]
Are you sure about 4? I believe I fixed that a few days after Heroics released

**Reply by Apathyxia** [reply]
I can confirm 4. Just yesterday pulling one pack pulled the 2 others, too.

**Reply by balleny** [reply]
yep, confirmed it today. 2 packs chaining / linked

**Reply by Gultask** [reply]
OH you mean the *other* skeletons :P 
Right, I'll fix it asap
